Makefile

```diff
@@ -36,6 +37,11 @@ else
 endif
 $(info STAGINGIMAGE is $(STAGINGIMAGE))
 
+BIN_DIR=bin
```

Comment:
You can do this with just BINDIR?=bin https://www.gnu.org/software/make/manual/html_node/Flavors.html#Flavors

Reply:
Done